sdk/core/Azure.Core.Experimental/api/Azure.Core.Experimental.netstandard2.0.cs

```diff
@@ -5,24 +5,27 @@ namespace Azure.Core
     {
         private readonly object _dummy;
         private readonly int _dummyPrimitive;
-        public BinaryData(System.ReadOnlyMemory<byte> data) { throw null; }
+        public BinaryData(System.ReadOnlySpan<byte> data) { throw null; }
         public BinaryData(string data) { throw null; }
-        public BinaryData(string data, System.Text.Encoding encoding) { throw null; }
-        public System.ReadOnlyMemory<byte> AsBytes() { throw null; }
+        public System.ReadOnlyMemory<byte> Bytes { get { throw null; } }
         public System.Threading.Tasks.ValueTask<T> DeserializeAsync<T>(Azure.Core.ObjectSerializer serializer, System.Threading.CancellationToken cancellationToken = default(System.Threading.CancellationToken)) { throw null; }
+        public System.Threading.Tasks.ValueTask<T> DeserializeAsync<T>(System.Threading.CancellationToken cancellationToken = default(System.Threading.CancellationToken)) { throw null; }
```

Comment:
Is there a reason for Async APIs on this type? All the data is in memory, i.e. no I/O will ever happen.

Reply:
The entire service registry story where we can fetch things as part of deserialization.